specs/tables.md

```diff
@@ -202,6 +202,20 @@ From this table, the following columns contain values using the RLC encoding:
 - ValuePrev
 - ValueCur
 
+## Keccak Table
+
+See [tx.py](src/zkevm_specs/tx.py)
+
+| IsEnabled | InputRLC   | InputLen | Output      |
+| --------- | ---------- | -------- | ----------- |
+| 1         | $input_rlc | $input_length | $output_rlc |
```

Comment:
What about this?
```suggestion
| bool         | $input_rlc | $input_length | $output_rlc |
```

Reply:
I mention this because this allows 0 to disable the lookup when needed. Linkin 0 -> 0